proposals/exp-show-target.md

```diff
@@ -0,0 +1,214 @@
+│ 551082e        │ Nov 15, 2020 │       - │ 0.2           │ 20170428     │ 1500                   │ 2                │ 20170428   │ 10                 │ -                  │ - │
+│ 15bef96        │ Nov 15, 2020 │       - │ 0.2           │ 20170428     │ 1500                   │ 2                │ 20170428   │ 10                 │ -                  │ - │
+└────────────────┴──────────────┴─────────┴───────────────┴──────────────┴────────────────────────┴──────────────────┴────────────┴────────────────────┴────────────────────┴───┘
+```
+
+This table includes parameters and metrics from the `evaluate` stage and all
+stages upstream of it.  It's possible to also have a `--single-item` argument to
+exclude upstream changes, but this is out of scope for now.
+
+# How We Teach This
+
+See [Detailed design](#detailed-design) for a proposed help message.
+
+This is consistent with other commands, and it's only one additional argument to
+an existing command, so it likely doesn't require much teaching. It might be
+necessary to explicitly add that parameters and metrics from upstream stages
+will be shown.
+
+# Drawbacks
+
+This proposal may require changes to how DVC collects and tracks parameters and
+metrics. Implementation could break existing functionality, and it is
+inconsistent with the current behaviors of `dvc params`, `dvc metrics`, and `dvc
+exp diff` commands.
+
+Further, this proposal adds to an already crowded set of `dvc exp show` options,
+and there is no single way to show parameters and metrics that's going to work
+for every use case. For example, this proposal does nothing to filter out
+parameters for upstream stages, which may frequently be irrelevant if only the
+stages near the end of the pipeline change during experimentation.
+
+# Alternatives
+
+There are many alternative methods to refine the output of `dvc exp show`:
+
+* Current functionality: `exp show` already has options to include/exclude any
+  column, so the flexibility to get the same output as suggested here already
+  exists.
+* Expand the include/exclude options to accept wildcard/glob/regex arguments for
+  more flexibility to exclude an entire file or section of a file like
+  `exclude-params=params.json:local_config*`.
+* Save configuration options for `dvc exp show` so they can be reused and
+  modified without typing out all options at the command line each time.
+* Add a tsv/csv output format option in `dvc exp show` to enable users to
+  manipulate the table with other tools.
+
+This proposal does not preclude any of these options. The current output
+will continue to be available except for untracked parameters. The other
+alternatives may still be implemented.
+
+Each of the alternatives provide flexibility to modify the `dvc exp show`
+outputs, which is important but not the goal of this proposal. Instead, the
+proposed feature uses the information DVC has to make smart decisions about what
```

Comment:
Sounds like this def. covers the discussion about default behavior then. Which I think is good (but may need clarification earlier on in the doc).